### PR TITLE
Add Ecovacs sensors

### PR DESCRIPTION
**Pull request by edenhaus**
## Proposed change

SSIA

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/108686
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by jbouwh — APPROVED** [review]
LGTM,
Thnx @edenhaus 👍

File: source/_integrations/ecovacs.markdown [hunk 1]
```diff
@@ -33,7 +34,20 @@ Additionally, **depending on your model**, the integration provides the followin
 
 - **Binary sensor**: 
   - `Mop attached`: On if the mop is attached. Note: If you do not see the state change to `Mop attached` in Home Assistant, you may need to wake up the robot in order to push the state change. Some models report an entity state change only if the overall status of the vacuum has changed. For example, if the overall state changes from `docked` to `cleaning`. 
-
+- **Sensor**:
+  - `Error`: The error code and a description of the error. `0` means no error. Disabled by default
+  - `Lifespan`: For each component a entity with the remaining lifespan will be created
```
Comment (MartinHjelmare) [inline source/_integrations/ecovacs.markdown:39]: an entity

File: source/_integrations/ecovacs.markdown [hunk 2]
```diff
@@ -33,7 +34,20 @@ Additionally, **depending on your model**, the integration provides the followin
...
+  - `Network`: The follwing network realted entities will be created. All disabled by default
```
Comment (MartinHjelmare) [inline source/_integrations/ecovacs.markdown:40]: The following network related entities

File: source/_integrations/ecovacs.markdown (hunk 1 above)
Comment (jbouwh) [reply source/_integrations/ecovacs.markdown:39]: See https://github.com/home-assistant/home-assistant.io/pull/31020

File: source/_integrations/ecovacs.markdown (hunk 2 above)
Comment (jbouwh) [reply source/_integrations/ecovacs.markdown:40]: See https://github.com/home-assistant/home-assistant.io/pull/31020

**Pull request merged**